include/couch_eunit.hrl

```diff
@@ -31,6 +31,11 @@
     filename:join([?BUILDDIR(), "src", "couch", "test", "fixtures"])).
 -define(TEMPDIR,
     filename:join([?BUILDDIR(), "tmp", "tmp_data"])).
+-define(ABS_PATH(File),
+    filename:join([
+        filename:dirname(
+            filename:dirname(
+                filename:dirname(filename:absname(File)))), File])).
```

Comment:
I lost myself here counting what the actual path here will be. Is there way to make it better?